src/Microsoft.AspNet.Mvc.ViewFeatures/ValidateAntiForgeryTokenAttribute.cs

```diff
@@ -9,6 +9,14 @@
 
 namespace Microsoft.AspNet.Mvc
 {
+    /// <summary>
+    /// Specifies that the class or method that this attribute is applied validates the anti-forgery token.
```

Comment:
Bit wonky because the class or method doesn't do the validation. Suggest `Specifies that MVC should validate the anti-forgery token prior to invoking the associated method or a method in the associated class.`